rel/overlay/etc/default.ini

```diff
@@ -189,27 +210,29 @@ database_prefix = userdb-
 [httpd]
 port = {{backend_port}}
 bind_address = 127.0.0.1
-authentication_handlers = {couch_httpd_auth, cookie_authentication_handler}, {couch_httpd_auth, default_authentication_handler}
-secure_rewrites = true
-allow_jsonp = false
+
 ; Options for the MochiWeb HTTP server.
 ;server_options = [{backlog, 128}, {acceptor_pool_size, 16}]
 ; For more socket options, consult Erlang's module 'inet' man page.
 ;socket_options = [{recbuf, undefined}, {sndbuf, 262144}, {nodelay, true}]
 socket_options = [{sndbuf, 262144}]
-enable_cors = false
-enable_xframe_options = false
+
+; These settings were moved to [chttpd]
+;authentication_handlers =
+;secure_rewrites =
+;allow_jsonp =
+;enable_cors =
+;enable_xframe_options =
 ; CouchDB can optionally enforce a maximum uri length;
-; max_uri_length = 8000
-; changes_timeout = 60000
-; config_whitelist = 
-; max_uri_length = 
-; rewrite_limit = 100
-; x_forwarded_host = X-Forwarded-Host
-; x_forwarded_proto = X-Forwarded-Proto
-; x_forwarded_ssl = X-Forwarded-Ssl
-; Maximum allowed http request size. Applies to both clustered and local port.
-max_http_request_size = 4294967296 ; 4GB
+; max_uri_length =
+; changes_timeout =
+; config_whitelist =
+; rewrite_limit =
+; x_forwarded_host =
+; x_forwarded_proto =
+; x_forwarded_ssl =
+;Maximum allowed http request size. Applies to both clustered and local port.
```

Comment:
this one was removed too I think, we can remove its comment here

Reply:
deleted